SConstruct

```diff
@@ -5,8 +5,11 @@ import platform
 import subprocess
 import time
 import errno
+import SCons.Warnings
```

Comment:
(pylint-no-name-in-module) No name 'Warnings' in module 'SCons'
(pylint-import-error) Unable to import 'SCons.Warnings'